packages/cli/src/services/playwright-config.ts

```diff
@@ -46,7 +47,7 @@ export class PlaywrightConfig {
   projects?: PlaywrightProject[]
   files: Set<string>
 
-  constructor(dir: string, playwrightConfig: any) {
+  constructor(dir: string, playwrightConfig: any, configFileName: string) {
     this.projectName = ''
```

Comment:
playwrightConfigPath: string

this.dir = path.dirname(playwrightConfigPath)
this.filePath = playwrightConfigPath